hack/verify-govet.sh

```diff
@@ -23,7 +23,8 @@ FAILURE=false
 test_dirs=$(find_files | cut -d '/' -f 1-2 | sort -u)
 for test_dir in $test_dirs
 do
-  go tool vet -shadow=false $test_dir
+  # composites are disabled because fielderrors.ValidationErrorList{...} is incorrectly flagged
```

Comment:
ValidationErrorList is gone in the rebase, and isn't needed any more to prefix, so I'll back this out